node/src/reactor/main_reactor.rs

```diff
@@ -202,6 +202,7 @@ pub(crate) struct MainReactor {
     node_startup_instant: Instant,
 
     finality_signature_creation: bool,
+    prevent_validator_shutdown: bool,
```

Comment:
I don't remember if this wasn't intentionally removed. We should make sure that reintroducing it isn't a mistake.

Reply:
I think we're good here. This feature was initially hardcoded and then removed, and then re-introduced as a configurable option [here](https://github.com/casper-network/casper-node/pull/4484).